tools/serverpod_cli/lib/src/generator/dart/library_generators/server_test_tools_generator.dart

```diff
@@ -3,6 +3,7 @@ import 'package:path/path.dart' as p;
 import 'package:serverpod_cli/analyzer.dart';
```

Comment:
Sorted members in this file but the only change is the `_buildWithServerpodFunction` method.

Reply:
Suggestion: Please do these type of changes in pure Refactor commits to separate code changes from refactors.

Reply:
100% agree, this was my mistake! It was a fairly small change so I thought it could be acceptable this time.